src/nodedef.h

```diff
@@ -125,9 +125,25 @@ enum LeavesStyle {
 	LEAVES_OPAQUE,
 };
 
+enum AutoScale {
```

Comment:
enum AutoScale : u8

Reply:
surely any reasonable compiler would optimise this?